pubspec.yaml

```diff
@@ -5,17 +5,21 @@ homepage: https://github.com/Workiva/over_react_test/
 authors:
   - Workiva UI Platform Team <uip@workiva.com>
 environment:
-  sdk: ">=1.24.2 <2.0.0"
+  sdk: ">=1.24.2 <3.0.0"
 dependencies:
   js: ^0.6.1
   matcher: ^0.12.1+4
   over_react: ^1.25.0
```

Comment:
This lower bound needs to be updated to the version of over_react that adds in the `.meta` constants that are used in this package (e.g., `DomPropsMixin.meta`).

Reply:
ah yea, good call!